01_fib/brvishnu92/main_test.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMain(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote(`1
+1
+2
+3
+5
+8
+13
+`)
+	actual := strconv.Quote(buf.String())
+	if expected != actual {
+		t.Errorf("Expected %v, got %v ", actual, expected)
+	}
+}
+
+func TestMainFib(t *testing.T) {
```

Comment:
Consider rename it to TestFib as you are only testing fib function.

Reply:
Oh yeah, thanks for that. I have updated it!!